build.sbt

```diff
@@ -174,6 +174,7 @@ lazy val storage = (project in file("storage"))
       // User can provide any 2.x or 3.x version. We don't use any new fancy APIs. Watch out for
       // versions with known vulnerabilities.
       "org.apache.hadoop" % "hadoop-common" % "3.3.1" % "provided",
+      "org.apache.hadoop" % "hadoop-aws" % "3.3.1" % "provided",
```

Comment:
@zsxwing - are you okay with including this as a provided dependency?

Reply:
Yep. This should be fine. A `provided` dependency will not be pulled automatically when a user depends on `delta-storage`.